<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportView</class>
 <widget class="QWidget" name="ReportView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>20</y>
     <width>248</width>
     <height>183</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>250</y>
     <width>248</width>
     <height>461</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <item>
    <property name="text">
     <string>0612P170178</string>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
   </item>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>701</width>
     <height>691</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::NoSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <column>
    <property name="text">
     <string>Datum</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Šarže</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Program</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Tisková úloha</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Operátor</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nastavení</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Pořadí</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>220</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Šarže:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
